src/main/resources/view/AppointmentListCard.fxml

```diff
@@ -9,7 +9,7 @@
 <?import javafx.scene.layout.VBox?>
 
 <?import javafx.scene.layout.Region?>
-<BorderPane id="borderPane" fx:id="borderPane" maxHeight="175" maxWidth="Infinity" minHeight="100" minWidth="300" prefHeight="160" prefWidth="400" xmlns="http://javafx.com/javafx/8.0.172-ea" xmlns:fx="http://javafx.com/fxml/1" styleClass="appt_card">
+<BorderPane id="borderPane" fx:id="borderPane" maxHeight="175" maxWidth="Infinity" minHeight="100" minWidth="300" prefHeight="160" prefWidth="300" xmlns="http://javafx.com/javafx/8.0.172-ea" xmlns:fx="http://javafx.com/fxml/1" styleClass="appt_card">
     <padding>
         <Insets bottom="5" left="20" right="5" top="5" />
```

Comment:
i realised that the string will still overflow when it hits the maximum number of characters and when the scroll bar appears, can you help change the left and right margin to 5 instead?